tests/packet_trimming/base_packet_trimming.py

```diff
@@ -399,18 +377,20 @@ def test_trimming_counters(self, duthost, ptfadapter, test_params, trim_counter_
                           "Trim sent counter on switch level is not equal to the sum of trim sent counter on port "
                           "level")
 
+        trim_queue = PacketTrimmingConfig.get_trim_queue(duthost)
```

Comment:
Please also update tests/generic_config_updater/test_packet_trimming_config_symmetric.py and tests/generic_config_updater/test_packet_trimming_config_asymmetric.py